src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
-body {
-  margin: 0;
+@import "./utils/variables";
+
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  &__body {
+    margin: 0;
+  }
+}
+
+.body {
+  font-family: "Arial", sans-serif;
+
+  &__calendar {
+    margin: 0 auto;
+  }
+}
+
+.calendar {
+  display: flex;
+  width: calc((#{$sizeCellDay} * 7) + (#{$sizeGap} * (7 - 1)));
+  padding-top: 10px;
+
+  flex-wrap: wrap;
+  gap: $sizeGap;
+
+  &__day {
+    display: flex;
+    width: $sizeCellDay;
+    height: $sizeCellDay;
+    border: 1px solid black;
+
+    justify-content: center;
+    align-items: center;
+
+    background-color: #eee;
+    cursor: pointer;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+
+    @for $dayOfMonth from 1 through 31 {
+      &:nth-child(#{$dayOfMonth}) {
+        &::before {
+          content: "#{$dayOfMonth}";
+          font-size: 30px;
+        }
```

Comment:
It is better to change only styles that are actually different when using scss `@for` and similar methods.

So it is better to move `font-size` to, say, &__day, instead setting it for each pseudo-element